src/main/java/org/jenkinsci/maven/plugins/hpi/hosting/PreferLTS.java

```diff
@@ -0,0 +1,35 @@
+package org.jenkinsci.maven.plugins.hpi.hosting;
+
+import org.apache.maven.project.MavenProject;
+import org.jenkinsci.maven.plugins.hpi.hosting.VerificationMessage.Severity;
+
+public class PreferLTS implements VerificationRule {
+	@Override
+	public VerificationMessage validate(MavenProject project) {
+		MavenProject jenkinsParent = findJenkinsParentPom(project);
+		if (jenkinsParent == null) {
+			return new VerificationMessage(
+					"Your plugin doesn't seem to have the Jenkins parent pom in its POM hierarchy, "
+							+ "it's generally not a normal situation",
```

Comment:
I'd end the previous line with a period and then say something like "Jenkins plugins generally inherit from the Jenkins parent POM."